browser/ui/tabs/test/BUILD.gn

```diff
@@ -7,14 +7,13 @@ source_set("browser_tests") {
   testonly = true
   defines = [ "HAS_OUT_OF_PROC_TEST_RUNNER" ]
 
-  sources = [
-    "brave_tab_strip_model_browsertest.cc",
-  ]
+  sources = [ "brave_tab_strip_model_browsertest.cc" ]
 
   deps = [
     "//brave/common:pref_names",
     "//chrome/app:command_ids",
     "//chrome/browser",
+    "//chrome/browser/profiles:profile",
```

Comment:
this could help with a lot of the existing circular deps with check_includes = false